src/WireMock.Net/RegularExpressions/RegexGuid.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace WireMock.RegularExpressions
```

Comment:
General: you can just write comments / code on 1 line, no need to break at 80 characters.